integration-tests/main-command-mode/pom.xml

```diff
@@ -45,6 +45,10 @@
             <groupId>org.apache.camel.quarkus</groupId>
             <artifactId>camel-quarkus-timer</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-vertx-web</artifactId>
```

Comment:
I guess the vertx web client is needed by the test harness.

Reply:
Correct.